app/views/admin/topical_event_about_pages/edit.html.erb

```diff
@@ -1 +1,23 @@
-<%= render partial: 'form', locals: { page_title: "Edit page about #{@topical_event.name}", url: admin_topical_event_topical_event_about_pages_path(@topical_event, @topical_event_about_page) } %>
+<% content_for :page_title, "Edit #{@topical_event}" %>
+<% content_for :title, "Edit #{@topical_event}" %>
```

Comment:
I think the `page_title` and `title` content need to be "Edit page about #{@topical_event}" (as in Bootstrap version)